extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -215,7 +228,8 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     if (method === 'reply') {
       const header = `<div ${dirAttr}>On ${dateStr}, ${from ?? ''} wrote:</div>`;
       const sanitizedQuote = Xss.htmlSanitize(header + escapedText);
-      return `<blockquote ${dirAttr}>${sanitizedQuote}</blockquote>`;
+      const thunderbirdClass = Catch.isThunderbirdMail() ? 'class="height-0"' : '';
+      return `<blockquote ${thunderbirdClass} ${dirAttr}>${sanitizedQuote}</blockquote>`;
```

Comment:
also for this `height-0` class - I tried to remove it, and didn't notice any changes in the previous message quote rendering. maybe it's not needed?

Reply:
It fixes the blockquote scroll when in fullscreen where without it, it will make the secure compose window extremely long when quoted email contains a lot of newlines . eg long emails. You could open long HTML emails and you'll notice the UI bug.

Reply:
Ah, I see, thanks for clarification!

Reply:
No worries at all. thank you for checking this extensively.